src/library_pthread.js

```diff
@@ -185,7 +185,7 @@ var LibraryPThread = {
           delete Module['canvas'];
 #endif
 
-          postMessage({ cmd: 'exit' });
+          postMessage({ cmd: 'exit', threadId: tb });
```

Comment:
Passing `threadId: tb` here is not needed, in the `onmessage` the receiver will know which thread the worker was hosting, via `worker.pthread.thread` field.